src/access/accessom_coupler/ocean_solo.F90

```diff
@@ -454,12 +413,25 @@ program main
   call mpp_clock_begin(main_clock)
   do nc=1, num_cpld_calls
 
-     call external_coupler_sbc_before(Ice_ocean_boundary, Ocean_sfc, nc, dt_cpld)
+     call external_coupler_sbc_before(Ice_ocean_boundary, Ocean_sfc, Atm_fields, nc, dt_cpld)
 
+     ! Potentially override fields from the data_table
      call mpp_clock_begin(override_clock)
-     call ice_ocn_bnd_from_data(Ice_ocean_boundary)
+     Time_next = Time + Time_step_coupled
```

Comment:
I see the comment in `ice_ocn_bnd_from_data` probably covers this.

Reply:
I did this because [all of the existing data override](https://github.com/mom-ocean/MOM5/blob/59e0aad14896027d4c11a936b936a0325e77dbcf/src/accessom_coupler/ocean_solo.F90#L577-L602) is done using fields at `Time + Time_step_coupled` and I wanted to be consistent with that.